api/api.go

```diff
@@ -89,6 +60,31 @@ func packageHandler(w http.ResponseWriter, r *http.Request) {
 	_, _ = w.Write(stringified)
 }
 
+func resolveDependencies(pkg *NpmPackageVersion, versionConstraint string) error {
+	pkgMeta, err := fetchPackageMeta(pkg.Name)
+	if err != nil {
+		return err
+	}
+	concreteVersion, err := highestCompatibleVersion(versionConstraint, pkgMeta)
+	if err != nil {
+		return err
```

Comment:
[review] would like to see this error get partially rewrapped to include the package name as well for easier tracing.